docs/index.rst

```diff
@@ -1,107 +1,24 @@
 .. _cpdocker_intro:
 
-Installing using Docker
-=======================
+Docker
```

Comment:
Didn't we just recently change this to `Installing using Docker`?

Reply:
yeah, we did. i was thinking the old title made sense with this reorganization, but on 2nd thought, it think not (i'll change back to `Installing using Docker`): 
![image](https://user-images.githubusercontent.com/11722533/33405167-23e8c97c-d51c-11e7-827a-36cc90ff77db.png)